services/QuillLMS/lib/tasks/attach_new_topics_to_activities.rake

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
```

Comment:
Not a huge deal, but you could combine these two `rake` tasks into a single file given that you're already sharing the namespace.  A single `topics.rake` file just makes things slightly more compact, but it's no big deal if you don't do so.